### PR TITLE
L2-479: Validate account id client

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

User sees an invalid account error on blur.

# Changes

* Use same nns-js validation on blur for the Address component.

# Tests

* Update broken tests.
* Add new test for new use case of error appearing on blur.

### CODE REVIEW

**Comment by lmuntaner** [comment]
UX video:

https://user-images.githubusercontent.com/4550653/169457581-3c209a19-cdac-4073-986f-d7ba631d9619.mov

**Comment by lmuntaner** [comment]
@peterpeterparker @mstrasinskis please review

I know that @peterpeterparker was not a fan of the layout shift, but with Mischa we agreed that this was the best bad solution for now.

**Review by bitdivine — APPROVED** [review]
LGTM, thanks!

File: frontend/svelte/src/lib/components/ui/Input.svelte [hunk 1]
```diff
@@ -28,7 +29,7 @@
   $: placeholder = translate({ labelKey: placeholderLabelKey });
 </script>
 
-<div class={`input-block ${theme}`} class:disabled>
+<div class={`input-block ${theme} `} class:disabled class:withErrorMessage>
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/components/ui/Input.svelte:32]: @lmuntaner can you style it using `:global` selector in `InputWithError`?

if not possible, please do not use a property for styling purpose only but rather a style, like CSS variable

Comment (lmuntaner) [reply frontend/svelte/src/lib/components/ui/Input.svelte:32]: ok

Comment (peterpeterparker) [reply frontend/svelte/src/lib/components/ui/Input.svelte:32]: https://github.com/dfinity/nns-dapp/pull/891

**Pull request merged**